src/material/core/tokens/m2/mat/_select.scss

```diff
@@ -17,6 +17,7 @@ $prefix: (mat, select);
   $on-surface: if($is-dark, #fff, #000);
 
   @return (
+    list-item-selected-container-color: rgba($on-surface, if($is-dark, 0.08, 0.04)),
```

Comment:
I had to lower the elevation for autocomplete in M3. I'm guessing we need something similar for select?